actors/builtin/paych/paych_actor.go

```diff
@@ -157,6 +159,11 @@ func (pca Actor) UpdateChannelState(rt vmr.Runtime, params *UpdateChannelStatePa
 		rt.Abortf(exitcode.ErrIllegalArgument, "voucher signature invalid: %s", err)
 	}
 
+	pchAddr := rt.Message().Receiver()
+	if pchAddr != sv.PaymentChannelAddr {
+		rt.Abortf(exitcode.ErrIllegalArgument, "voucher payment channel address does not match receiver")
```

Comment:
Include `sv.PaymentChannelAddr` in the message

Reply:
This is done.